modules/javafx.media/src/main/native/gstreamer/3rd_party/libffi/include/ffi_cfi.h

```diff
@@ -71,6 +71,7 @@
 # define cfi_personality(enc, exp)
 # define cfi_lsda(enc, exp)
 # define cfi_escape(...)
+# define cfi_window_save
```

Comment:
The libffi 3.5.2 source contains a definition of `cfi_window_save` under `#ifdef HAVE_AS_CFI_PSEUDO_OP` as well. Could you please check.